scripts/asr_language_modeling/ngram_lm/kenlm_utils.py

```diff
@@ -17,19 +17,32 @@
 Utility methods to be used for training N-gram LM with KenLM in train_kenlm.py
 """
 
+CHUNK_SIZE = 8192
+CHUNK_BUFFER_SIZE = 512
+
+import argparse
+import gzip
 import json
 import os
 
 import numpy as np
+import torch
 from joblib import Parallel, delayed
 from tqdm.auto import tqdm
 
+import nemo.collections.asr as nemo_asr
+from nemo.collections.asr.parts.submodules.ctc_beam_decoding import DEFAULT_TOKEN_OFFSET
+from nemo.collections.asr.parts.utils.transcribe_utils import PunctuationCapitalization
+from nemo.collections.common.tokenizers.sentencepiece_tokenizer import SentencePieceTokenizer
+from nemo.utils import logging
+
 # List of the supported models to be used with N-gram LM and beam search decoding
 SUPPORTED_MODELS = {
     'EncDecCTCModelBPE': 'subword',
     'EncDecCTCModel': 'char',
     'EncDecRNNTBPEModel': 'subword',
     'EncDecRNNTModel': 'char',
+    'EncDecHybridRNNTCTCBPEModel': 'subword',
```

Comment:
Would you please add EncDecHybridRNNTCTCModel?

Reply:
Done